data/tddconfig-correctness-test-both.json

```diff
@@ -4,8 +4,14 @@
   "demul_block_size": 40,
   "bs_radio_num": 8,
   "ue_radio_num": 8,
-  "modulation": "64QAM",
-  "Zc": 104,
+  "ul_mcs": {
+    "modulation": "64QAM",
+    "Zc": 104
+  },
+  "dl_mcs": {
+    "modulation": "64QAM",
+    "Zc": 102
```

Comment:
Is the value of Zc set to be smaller than uplink intentionally? Based on the 3GPP specs, 102 is not in the set of possible values of Zc (Table 5.3.2-1 in https://www.etsi.org/deliver/etsi_ts/138200_138299/138212/15.03.00_60/ts_138212v150300p.pdf)